media/css/base/mozilla-fxa-iframe.scss

```diff
@@ -0,0 +1,11 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#fxa {
+    opacity: 0;
+
+    &.loaded {
+        opacity: 1;
+    }
+}
```

Comment:
Guessing this is intended to be used on other FxA pages at a later date?

Reply:
I'm just following existing conventions here really. The `.less` version of this import is used by both `/firstrun/` and `/accounts/`. I'm guessing we'll want to do the same once `/firstrun/` is ported to Pebbles.

Reply:
👍